app/main.py

```diff
@@ -1,3 +1,49 @@
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: Union[int, float]) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Union[int, float, "Distance"]) -> "Distance":
```

Comment:
fix it and other places 

Article about this https://stackoverflow.com/questions/44664040/type-hints-with-user-defined-classes
```suggestion
    def __add__(self, other: Union[int, float, Distance]) -> Type[Distance]:
```

Reply:
<img width="1042" alt="Screenshot 2023-05-14 at 20 46 51" src="https://github.com/mate-academy/py-distance-class/assets/114707574/7b7de3a3-c438-46cd-8f1e-2af4003ac590">
Hi. While following your instructions I have PyCharm error: Expected type 'Type[Distance]', got 'Distance' instead in the __add__ function. 
Can I use the method described below instead? 

from __future__ import annotations

    def __add__(self, other: Union[int, float, Distance]) -> Distance: